knimin/templates/auth_sitebase.html

```diff
@@ -49,12 +57,23 @@
             {% end %}
         </ul>
 
+        <h3>Plate Mapper</h3>
+        <ul class="mainmenu">
+            {% if admin or db.has_access(current_user, ['Admin']) %}
```

Comment:
@qiyunzhu, I think @josenavas is referring to the fact that the if clause is checking for the exact same condition to be met:

Line 62:
{% if admin or db.has_access(current_user, ['Admin']) %}

Line 65:
{% if admin or db.has_access(current_user, ['Admin']) %}